### PR TITLE
(JP) Update doc about changing reference OS (#4062)

### PR DESCRIPTION
**Pull request by fisuda**
This PR updates Japnese documentation about changing reference OS.
It would be great if you could review tihs PR.

Thanks.

### CODE REVIEW

File: doc/manuals.jp/admin/install.md [hunk 1]
```diff
@@ -16,69 +11,34 @@
 <a name="introduction"></a>
 ## イントロダクション
 
-推奨される手順は、公式の [Docker Hub の Orion docker コンテナ](https://hub.docker.com/r/fiware/orion/) を使用してインストールすることです。
+推奨される手順は、公式の [Docker Hub の Orion docker コンテナ](https://hub.docker.com/repository/docker/telefonicaiot/fiware-orion) を使用してインストールすることです。詳細については、[このドキュメント](https://github.com/telefonicaid/fiware-orion/blob/master/docker/README.md)を参照してください。
```
Comment (fgalan) [inline doc/manuals.jp/admin/install.md:14]: Maybe better?

```suggestion
推奨される手順は、公式の [Docker Hub の Orion docker コンテナ](https://hub.docker.com/repository/docker/telefonicaiot/fiware-orion) を使用してインストールすることです。詳細については、[このドキュメント](https://github.com/telefonicaid/fiware-orion/blob/master/docker/README.jp.md)を参照してください。
```

File: doc/manuals.jp/admin/install.md [hunk 2]
```diff
@@ -16,69 +11,34 @@
...
-* オペレーティング・システム : CentOS/RedHat。リファレンス・オペレーティング・システムは CentOS 8.3.2011 ですが、それ以降の CentOS/RedHat 8.x バージョンでも動作します
-* データベース : MongoDB は、Orion Context Broker をインストールするホストと同じホストで実行するか、ネットワーク経由でアクセス可能な別のホストで実行する必要があります。推奨される MongoDB のバージョンは 4.4 です。古いバージョンでは動作しますが、推奨しません
-* RPM の依存関係 (これらのパッケージのいくつかは、公式の CentOS/RedHat リポジトリにはありませんが、EPEL にあります。EPEL リポジトリを設定する必要があります。<http://fedoraproject.org/wiki/EPEL> を参照してください) :
-    * contextBroker パッケージ (必須) は次のパッケージに依存しています : libstdc++, boost-thread, boost-filesystem, gnutls, libgcrypt, libcurl, openssl, logrotate, libuuid
+Docker hub で公式の Orion docker コンテナを使用してインストールする場合は、次のものが必要です:
 
-<a name="installation"></a>
-## インストール
+* [Docker](https://docs.docker.com/engine/install/)
+* [Docker compose](https://docs.docker.com/compose/install/) (not strictly needed, but strongly recommended)
```
Comment (fgalan) [inline doc/manuals.jp/admin/install.md:25]: "(not strictly needed, but strongly recommended)" should be translated?

File: scripts/managedb/csubs_control/README.md [hunk 3]
```diff
@@ -2,7 +2,7 @@ This directory includes some scripts to help to control context subscriptions co
 the same CLI arguments:
 
 * `-db <database>` (mandatory): to specify the DB to process
-* `--dry-run (optional): it avoids actual modification in the DB, just reporting
+* `--dry-run` (optional): it avoids actual modification in the DB, just reporting
```
Comment (fgalan) [inline scripts/managedb/csubs_control/README.md:5]: Thanks for the typo fixing!

File: doc/manuals.jp/admin/install.md (hunk 1 above)
Comment (fisuda) [reply doc/manuals.jp/admin/install.md]: Thank you for the comment. I fixed it.

File: doc/manuals.jp/admin/install.md (hunk 2 above)
Comment (fisuda) [reply doc/manuals.jp/admin/install.md]: I have overlooked it. I translated it.

**Review by fgalan — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**